tests/test_smoke.py

```diff
@@ -3417,6 +3417,26 @@ def test_skyserve_cancel(generic_cloud: str):
     run_one_test(test)
 
 
+@pytest.mark.serve
+@pytest.mark.no_kubernetes
+def test_skyserve_streaming(generic_cloud: str):
+    """Test skyserve with streaming"""
+    name = _get_service_name()
+    test = Test(
+        f'test-skyserve-streaming',
+        [
+            f'sky serve up -n {name} --cloud {generic_cloud} -y tests/skyserve/streaming/streaming.yaml',
+            _SERVE_WAIT_UNTIL_READY.format(name=name, replica_num=1),
+            f'{_SERVE_ENDPOINT_WAIT.format(name=name)}; '
+            'python3 tests/skyserve/streaming/send_streaming_request.py '
+            '--endpoint $endpoint',
```

Comment:
Should we have a way to detect any failure?

Reply:
Added. Now it will raise and abort the test. Thanks!